lib/std/rand/Romu192.zig

```diff
@@ -0,0 +1,141 @@
+// Website: romu-random.org
+// Reference paper:   http://arxiv.org/abs/2002.11331
+
+// important note: this RNG exposes all of it's state in the random sequences generated, so it is trivial to write a state prediction function.
+// it's use is recommended only in context where there is no risk of prediction, e.g. it is perfectly safe for a monte carlo simulation.
+
+const std = @import("std");
+const Random = std.rand.Random;
+const math = std.math;
+const Romu192 = @This();
+
+xState: u64,
+yState: u64,
+zState: u64, // set to nonzero seed
+
+pub fn init(init_s: u64) Romu192 {
+    var x = Romu192{ .xState = undefined, .yState = undefined, .zState = undefined };
+    x.seed(init_s);
+    return x;
+}
+
+pub fn random(self: *Romu192) Random {
+    return Random.init(self, fill);
+}
+
+fn next(self: *Romu192) u64 {
+    const xp = self.xState;
+    const yp = self.yState;
+    const zp = self.zState;
+    self.xState = 15241094284759029579 *% zp;
+    self.yState = yp -% xp;
+    self.yState = std.math.rotl(u64, self.yState, 12);
+    self.zState = zp -% yp;
+    self.zState = std.math.rotl(u64, self.zState, 44);
+    return xp;
+}
+
+pub fn seed_with_buf(self: *Romu192, buf: [24]u8) void {
```

Comment:
Same style suggestion here.
```suggestion
pub fn seedWithBuf(self: *Romu192, buf: [24]u8) void {
```